.eslintrc.js

```diff
@@ -360,6 +360,7 @@ module.exports = {
     'no-else-return': ['off'],
     'spaced-comment': 'off',
     'line-comment-position': ['error', {position: 'above'}],
+    'arrow-parens': 'off',
```

Comment:
This allows us to get rid of parenthesis around a single parameter in arrow functions, for example:
```
resources.some((resource) => ...)
```
becomes
```
resources.some(resource => ...)
```